auto_tests/toxav_many_test.c

```diff
@@ -163,15 +162,10 @@ void *call_thread(void *pd)
 
     c_sleep(30);
 
-    int16_t PCM[960];
-    uint8_t video_y[800 * 600];
-    uint8_t video_u[800 * 600 / 2];
-    uint8_t video_v[800 * 600 / 2];
-
-    memset(PCM, 0, sizeof(PCM));
-    memset(video_y, 0, sizeof(video_y));
-    memset(video_u, 0, sizeof(video_u));
-    memset(video_v, 0, sizeof(video_v));
+    int16_t *PCM = calloc(960, sizeof (*PCM));
```

Comment:
In the rest of the code we use sizeof(TypeName) instead of sizeof(*var). It might be good to keep it consistent here and in other places (if used differently) as well.